cmd/syft/cli/options/output_file.go

```diff
@@ -13,13 +17,13 @@ var _ interface {
 
 // Deprecated: OutputFile supports the --file to write the SBOM output to
 type OutputFile struct {
-	Enabled bool   `yaml:"-" json:"-" mapstructure:"-"`
-	File    string `yaml:"file" json:"file" mapstructure:"file"`
+	Enabled    bool   `yaml:"-" json:"-" mapstructure:"-"`
+	LegacyFile string `yaml:"-" json:"-" mapstructure:"legacyFile"`
```

Comment:
This is the same as the already deprecated `--file` option:
```
--file string              file to write the default report output to (default is STDOUT) (DEPRECATED: use: output)
```

This will break any user configuration that is using `file:` . Given that this is an ideal time to make this breaking change and there isn't a really good semantic equivalent for file cataloging configuration, I think this is the right time to make this breaking change.